juriscraper/pacer/attachment_page.py

```diff
@@ -121,15 +147,47 @@ def _get_document_number(self):
                 continue
         return None
 
+    def _decrement_attachment_index(self):
```

Comment:
Interesting....can you put a mega comment about what's going on here? Also, how do we know it's only those four courts?

Reply:
> Interesting....can you put a mega comment about what's going on here?

yeah...I'm guessing some of the district courts previously generated the attachment numbers's the same way as the bankruptcy courts...then switched over...but it appears the old attachment numbers remained the same

> Also, how do we know it's only those four courts?

oh, it's probably more, these are just the ones I've found so far

Reply:
comment added

Reply:
Would it be possible to remove the dict and detect if decrementing is needed? That'd work across all times and courts?

If not, is there a way to figure out the starting IDs for all the other courts so we can do this right?

Reply:
> Would it be possible to remove the dict and detect if decrementing is needed? That'd work across all times and courts?

Not that I could find.

> If not, is there a way to figure out the starting IDs for all the other courts so we can do this right?

Yeah, but it will take a while since there's a lot of these, maybe merge what we have for now and track down the others later?

I did some spot checking and it appears there's quite a few that have buggy offsets for early docket sequences:
```
dcd good
almd good
alnd good
alsd good
akd buggy
azd good
ared buggy
arwd good
cacd buggy
caed buggy
cand good
casd good
cod good
ctd buggy
ded good
flmd buggy
flnd buggy
flsd good
gamd buggy
gand good
gasd buggy
hid buggy
idd good
ilcd good
ilnd buggy
ilsd good
innd buggy
insd good
iand buggy
iasd buggy
ksd good
kyed buggy
kywd buggy
laed good
lamd good
lawd good
med good
mdd good
mad buggy
mied buggy
miwd buggy
mnd good
msnd buggy
mssd buggy
moed buggy
mowd good
mtd good
ned buggy
nvd buggy
nhd good
njd good
nmd good
nyed buggy
nynd buggy
nysd buggy
nywd buggy
nced buggy
ncmd buggy
ncwd good
ndd buggy
ohnd good
ohsd good
oked buggy
oknd good
okwd good
ord  buggy
paed good
pamd buggy
pawd good
rid good
scd good
sdd buggy
tned good
tnmd good
tnwd good
txed buggy
txnd buggy
txsd good
txwd unknown
utd buggy
vtd good
vaed buggy
vawd good
waed buggy
wawd buggy
wvnd good
wvsd good
wied buggy
wiwd unknown
wyd buggy
gud good
nmid good
prd good
vid good
```

Reply:
Added changeover sequences for everything I managed to find.